py-polars/polars/config.py

```diff
@@ -399,7 +399,34 @@ def set_ascii_tables(cls, active: bool | None = True) -> type[Config]:
 
     @classmethod
     def set_auto_structify(cls, active: bool | None = False) -> type[Config]:
-        """Allow multi-output expressions to be automatically turned into Structs."""
+        """
+        Allow multi-output expressions to be automatically turned into Structs.
+
+        Examples
+        --------
+        >>> df = pl.DataFrame({"v": range(10), "v2": range(10, 20)})
```

Comment:
Avoid using data that is truncated when displayed. A smaller example is just as clear.

```suggestion
        >>> df = pl.DataFrame({"v": [1, 2, 3], "v2": [4, 5, 6]})
```

Reply:
Good point, change made